amplify.yml

```diff
@@ -10,7 +10,8 @@ applications:
             - nvm use 16
             - node -v
             - export FLUTTER_HOME=${HOME}/sdks/flutter
-            - git clone -b stable --depth 1 https://github.com/flutter/flutter.git ${FLUTTER_HOME}
+            # pin to flutter v3.3.10
```

Comment:
Is this a long term fix?

Reply:
Yes. I messaged you offline. If you have any other questions, lets jump on a call.